Directory.Build.props

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-
+   <PropertyGroup>
```

Comment:
This shouldn't be needed. The latest version of gallery should already be using WinUI2.8.

Reply:
> This shouldn't be needed. The latest version of gallery should already be using WinUI2.8.

By default, lottie-react-native uses WinUI 2.6, so to override this, they recommend setting that property in the Directory.Build.props file.

Reply:
Hmm since the app uses WinUI2.8 I would expect that to take precedence over lottie. Can you confirm that the build doesn't work without this change? If that doesn't work, could you also try specifying WinUI 2.8.0 in the app's ExperimentalFeatures file. See here for info on that: https://github.com/microsoft/react-native-gallery/blob/c7e197a8b6b39733986fd9f3f993dfdc2a0492ec/windows/ExperimentalFeatures.props#L35

Reply:
I have recompiled the project a few times to test, and I have not had any problems with the compilation of the package or the project without the property <WinUI2xVersion>2.8.0</WinUI2xVersion>.